pkg/smokescreen/metrics/mocks.go

```diff
@@ -174,6 +75,31 @@ func (m *MockMetricsClient) GetCount(metric string, tags ...string) (uint64, err
 	return i, nil
 }
 
+// GetValues returns the values stored for a metric metric has been updated since the MockMetricsClient was
+// created. To support GetValues being called with or without tags for a given metric, tagged metrics
+// are counted twice: once for the untagged metric ("foo") and once for the metric with its tags
+// sorted("foo [a b c]").
```

Comment:
```suggestion
// GetValues returns the values stored for a metric that has been updated since the MockMetricsClient was
// created.
```
(The bit about double-counting seemed not quite right here.)

Reply:
Well, the "foo"/"foo [a b c" bit is still relevant, since this function does record values for both--I can update the language to get rid of the word "counting" though